### DISCUSSION TITLE
Mini playground for Fusion

### DISCUSSION THREAD

**Discussion by Entrivax** [pfusik/cito#115] [Show and tell]
Hello!

I tried to make a little playground for Fusion for the fun.
I started the project yesterday and the repository is available at https://github.com/Entrivax/fu-playground
And the current version is deployed at https://entrivax.github.io/fu-playground/

There is nothing really serious about it, I just liked to try and experiment on this kind of project.
There is bugs, the code is absolutely not clean, but for a first iteration, it works.

Have fun!

![image](https://github.com/fusionlanguage/fut/assets/1197915/e6920ea0-d3ad-4f6f-a31d-487c5e336579)

**Comment by pfusik** [comment]
Thanks! Were you aware of https://fusion-lang.org ? The repo is https://github.com/fusionlanguage/fusion-lang.org

**Reply by Entrivax** [reply]
Oooh! I didn't know, actually, thanks!

**Reply by Entrivax** [reply]
Maybe you should set the url of this repo to that! It would be easier, since this repo is the most known!

**Reply by pfusik** [reply]
What do you mean by "set the url of this repo to that" ?
I'm going to update README.md and perhaps [Getting Started](https://github.com/fusionlanguage/fut/blob/master/doc/getting-started.md).

**Reply by Entrivax** [reply]
In the settings of the repository, you can add a website:
![image](https://github.com/fusionlanguage/fut/assets/1197915/b2708775-c826-4505-aa56-55ff2b9df555)

**Reply by pfusik** [reply]
Done!
